ssl/t1_trce.c

```diff
@@ -1505,7 +1505,8 @@ void SSL_trace(int write_p, int version, int content_type,
 
             /* avoid overlapping with length at the end of buffer */
             if (msglen < (size_t)(SSL_IS_DTLS(ssl) ?
-	                 DTLS1_RT_HEADER_LENGTH : SSL3_RT_HEADER_LENGTH)) {
+                                  DTLS1_RT_HEADER_LENGTH :
+                                  SSL3_RT_HEADER_LENGTH)) {
```

Comment:
Odd... I think I recall suggesting
```
            if (msglen < (size_t)(SSL_IS_DTLS(ssl) ? DTLS1_RT_HEADER_LENGTH
                                                   : SSL3_RT_HEADER_LENGTH)) {
```
here and I think I recall it was formatted like that... Though it might be another similar occasion...

Reply:
Line 1508 was one of those mixed tab+space lines. After reindenting, it exceeded the 80 char limit, because it was aligned so far to the right. I'll just replace tab by four spaces and leave the rest as it was.